gravitino.exceptions.base.ConnectionFailedException
===================================================

.. currentmodule:: gravitino.exceptions.base

.. autoexception:: ConnectionFailedException